controller/argument_value_resolver.rst

```diff
@@ -110,6 +110,14 @@ type-hinted method arguments:
             ),
         ));
 
+
+.. tip::
+    To keep Doctrine's `@ParamConverter` enabled, simply use an interface instead of an object.
+    
+    For example, using a `App\Interface\CustomUserInterface` instead of your `App\Entity\User`
+    will allow you to inject the interface while still using `@ParamConverter` for entities.
```

Comment:
Are you trying to say that we shouldn't be coupled to and concrete class but rather interface?

Not sure people (maybe it is just me) add interfaces to their Entities.

Maybe I just misunderstand the point of this.